todo_react/todo-react/src/context/context.tsx

```diff
@@ -0,0 +1,46 @@
+import React, {ChangeEvent, useContext} from "react";
+import {ITodo} from "../models";
+
+interface TodosContextModel {
+    todos: ITodo[],
```

Comment:
`todos` and `todoList` are duplicates?

Reply:
todoList - is the initial state for todos, I moved in a separate variable for easier readable